images/fedora/f28/missing-docs

```diff
@@ -0,0 +1,22 @@
+acl
+bash
+chkconfig
+curl
```

Comment:
I think we need `dbus` here.

If you do `rpm --list -q dbus | grep man` inside a F28 toolbox, you'll see some `man` pages listed, but those files actually don't exist. If I recall correctly, the `dbus-daemon` package split from the `dbus` package in F29, which probably confused the script.

Reply:
Added.